Taskfile.yml

```
@@ -34,6 +34,8 @@ tasks:
     deps: [ setup-win, setup-unix ]
     cmds:
       - git config blame.ignoreRevsFile .git-blame-ignore-revs
+      - git submodule init
+      - git submodule update --recursive
```

I think this would be a valuable addition:
```suggestion
      - git submodule update --recursive
      - git config --global submodule.recurse true
```

---

https://stackoverflow.com/questions/53514521/what-are-the-drawbacks-to-setting-gits-submodule-recurse-config-option-to-true mentions some drawbacks to setting that globally. Let's set it locally (for this repo), but not globally as that could affect other projects on the dev's machine.

---

Commit ec2285b9 adds this suggestion but with `--local` instead of `--global`. (`--local` is the default, but that's not immediately obvious to someone reading the Taskfile unless they know the Git manual by heart, so I figure it's better to be explicit about what the command does).